exampleSite/content/docs/multilingual/content.md

```diff
@@ -0,0 +1,99 @@
++++
+fragment = "content"
+weight = 100
+
+title = "MultiLingual Mode"
+
+[sidebar]
+  sticky = true
++++
+
+> Syna supports creating Multilingual websites since v0.16.1. The implementation
+> only supports `page[.language code].md` method and not 
+> `[language code]/page.md` but we hope to add the second approach in the 
+> future.
+
+In order to make your website multilingual, you need to change your
+configuration so Hugo will generate new pages for translated content in your
+project. The way Multilingual mode works is that Hugo generates a new website
+for each language and add each page to the website of the same language. For 
+example if your website has two languages, English and German, and each page
+is translated for both languages, Hugo will generate two websites with 
+same pages but in two languages. Hugo will not however generate English pages
+for German website if those pages are not translated to German. For more 
+information on Hugo's Multilingual mode, please refer to [their documentation](https://gohugo.io/content-management/multilingual/).
```

Comment:
Don't we default to english fragments? We might be doing it differently than hugo here and with that fact the above comment might be confusing. What do you think?

Reply:
Fragments are used in pages regardless of language. Pages are generated for each language if the page has the translation suffix in their filename. For example `page.md` is default language page and `page.fr.md` is generated for French. However, that is just a semantics thing. Our fragments switch to the language based on Hugo's i18n for not provided text or built-in texts but you can enter any text for any language regardless of page translation.

Reply:
Ah that makes sense.